dns/ddclient/src/opnsense/mvc/app/models/OPNsense/DynDNS/DynDNS.xml

```diff
@@ -51,8 +51,7 @@
                         <dinahosting>dinahosting</dinahosting>
                         <dnsmadeeasy>DNS Made Easy (digicert)</dnsmadeeasy>
                         <dns-o-matic>DNS-O-Matic</dns-o-matic>
-                        <dnsexit2>DNSExit API</dnsexit2>
-                        <dnsexit>DNSExit Legacy</dnsexit>
+                        <dnsexit>DNSExit</dnsexit>
```

Comment:
you should either rebase or fix with a merge from master

Reply:
Yeah, sorry for the mess ... I always mess git up as soon as I try to roll something back :) I hope it's OK though ...

Reply:
no worries, I fixed it manually